risk-to-severity-mapping/sync_risk_to_severity.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python3
+"""
+GitGuardian Risk Score to Severity Sync Script
+This script fetches incidents, converts risk scores to severity levels,
+and updates incidents so they can be properly synced with ArmorCode.
+
+Schedule this script to run daily via cron or your preferred scheduler.
+"""
+
+import os
+import requests
+import time
+from typing import Dict, List, Optional
+from datetime import datetime
+from dotenv import load_dotenv
+load_dotenv()
+
+# Configuration
+API_KEY = os.environ.get("GITGUARDIAN_API_KEY")  # Set this environment variable
+API_BASE_URL = os.environ.get("GITGUARDIAN_API_URL", "https://api.gitguardian.com")
+DRY_RUN = os.environ.get("DRY_RUN", "false").lower() == "true"
```

Comment:
I suggest we have the script running by default in DRY_RUN mode and explicitly have the user set DRY_RUN to false when they are ready to run it.